ci/builder/Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR /scratch
 # The environment variables are necessary to convince `tzdata`'s install scripts
 # not to hang.
 # See: https://serverfault.com/questions/949991/how-to-install-tzdata-on-a-ubuntu-docker-image
-RUN apt-get update && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+RUN apt-get update --fix-missing && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

Comment:
How did this work previously without `--fix-missing` ? Which package is the problem?

Reply:
`--fix-missing` did unfortunately not help. `nodejs` is causing issues, see https://materializeinc.slack.com/archives/C01KV5PEZ9R/p1698828173147049.